MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2236,6 +2245,7 @@ private PartInUse getPartInUse(Part p) {
             p = ((MissingPart) p).getNewPart();
```

Comment:
I know this isn't your code, but we're trying to move away from single letter or abbreviated variable names. As you're doing a lot of work in this neck of the woods would you mind changing `p` to `part`?